src/tasks/WasmAppBuilder/EmccExec.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Build.Tasks;
+
```

Comment:
`namespace Microsoft.WebAssembly.Build.Tasks;`

Reply:
And `#nullable enable`.

Reply:
Replaced with `OnError` msbuild feature